config/state_machine/configs/00_moj/50_offender_sar_complaint/20_offender_sar_complaint_standard_workflow.yml

```diff
@@ -17,9 +17,8 @@
                 preview_cover_page:
                 edit_case:
                 assign_to_team_member:
-                  if: Workflows::Predicates#responder_is_member_of_assigned_team_and_not_assigned?
+                  if: Workflows::Predicates#responder_is_not_assigned?
                 reassign_user:
-                  if: Workflows::Predicates#responder_is_member_of_assigned_team_and_assigned?
```

Comment:
Probably being dim, but I can't figure out why there were two different predicate methods used for the conditional on reassign_user - there's this one and `responder_is_member_of_assigned_team?` - do you understand why?

We've taken out all the predicate methods for `reassign_user`in this workflow anyway, so I guess it's moot!

Reply:
It's a fair question. There was a check on whether the was already a user assigned for 2 of the states (`legal_proceedings_ongoing` and `to_be_assessed`), but not for the others. 

I couldn't find a good reason for it, and as there shouldn't be a way for a case to be created without someone assigned, I think it is safe to not check that now anyway.